integrationExamples/mass/index.html

```diff
@@ -0,0 +1,110 @@
+<html>
+  <head>
+      <script>
+        window.massConfig = {
+          inskin: {
+            default: {
+              plr_PubfileId: '000000/mass'
+            }
+          }
+        };
+
+        var PREBID_TIMEOUT = 3300;
+  
+        var adUnits = [{
+          code: 'div-gpt-ad-1460505748561-0',
+          mediaTypes: {
+            banner: {
+              sizes: [[300, 250], [300, 600]],
+            }
+          },
+          bids: [{
+            bidder: 'ix',
+            params: {
+              siteId: '123456',
+              size: [300, 250]
+            }
+          }]
+        }];
+  
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+      </script>
+
+      <script>
+        pbjs.que.push(function() {
+          pbjs.setConfig({
+            mass: {
+              enabled: true,
+              bootloaderUrl: 'https://cdn.massplatform.net/bootloader.js',
```

Comment:
it was requested at js meeting today this changes to renderUrl to more closely align with other naming conventions. Also if this isn't locked to a version, it should be in the module config, not a default. All examples and documentation are welcome to point to it, but the publisher being required to explicitly name it in their config makes the committee more comfortable with it.